lib/api.js

```diff
@@ -882,6 +895,7 @@ function makeWrappedRunner (run, callback) {
  * @param {object}  [opts] - options
  * @param {boolean} [opts.enabled=true] - enable tracing
  * @param {boolean} [opts.collectBacktraces=false] - collect backtraces
+ * @param {boolean} [opts.forceNewTrace] - force a new trace, ignoring any existing context (but not xtrace)
```

Comment:
is there a default for opts.forceNewTrace ?

Reply:
false and i should have noted so. thanks.